distributed/utils.py

```diff
@@ -972,47 +972,6 @@ def read_block(f, offset, length, delimiter=None):
     return bytes
 
 
-def ensure_bytes(s):
```

Comment:
Instead of directly removing `distributed.utils.ensure_bytes`, I'd prefer to deprecate like we did with some similar utilities in https://github.com/dask/distributed/pull/4966. I wouldn't be surprised to see other projects using `distributed.utils.ensure_bytes` and it'd be good to give them a heads up before removing.

Reply:
Thanks for the feedback. Agreed. Have pushed a wrapper function that raises this warning.